x-pack/plugins/ml/server/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { HomeServerPluginSetup } from 'src/plugins/home/server';
+import { PluginSetupContract as FeaturesPluginSetup } from '../../features/server';
+import { LicensingPluginSetup } from '../../licensing/server';
+import { SpacesPluginSetup } from '../../spaces/server';
+
+export const PLUGIN_ID = 'ml';
+
+export interface LicenseCheckResult {
+  isAvailable: boolean;
+  isActive: boolean;
+  isEnabled: boolean;
+  isSecurityDisabled: boolean;
+  status?: string;
+  type?: string;
+}
+
+export interface PluginsSetup {
+  cloud: any;
```

Comment:
this should be `CloudSetup`

Reply:
609be5c4c218d1feed4b249158c438ce7c30f99d